tests/integration/helpers.bash

```diff
@@ -18,7 +18,7 @@ HELLO_IMAGE="$TESTDATA/hello-world.tar"
 HELLO_BUNDLE="$BATS_TMPDIR/hello-world"
 
 # CRIU PATH
-CRIU="$(which criu)"
+CRIU="$(which criu)" || true
```

Comment:
In my understanding, the logic is `(CRIU="$(which criu)") || true` which will always set `$?=0`. If I don't do this, once which criu failed, `bats` will quit (`set -e`)